File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.js
function solution(s) {
    let total = 0;
    let last = 0;
    
    for (const char of s.split(' ')) {
        if (char === 'Z') {
            total -= last;
            last = 0;
        } else {
            total += parseInt(char);
            last = parseInt(char);
        }
    }
    
    return total;
}

File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.py
def solution(s):
    s_list = s.split(" ")
    total = 0
    last = 0
    
    for char in s_list:
        if char == 'Z':
            total -= last
            last = 0
        else:
            total += int(char)
            last = int(char)
    
    return total